src/bp-search/classes/class-bp-search-bbpress-forums-topics.php

```diff
@@ -173,7 +173,9 @@ function ( $group_id ) {
 			);
 
 			$forum_id_in   = implode( ',', $forum_ids );
-			
+			if ( empty( $forum_id_in ) ) {
+				$forum_id_in = 0;
+			}
```

Comment:
@kishanjasani Below solution will be an optimized way to solve this problem [ As `implode` function is taking more execution time then assign value so that should be in condition ]
```
$forum_id_in   = 0
if ( ! empty( $forum_id_in ) ) {
	$forum_id_in   = implode( ',', $forum_ids );
}
```